README.md

```diff
@@ -48,7 +48,7 @@ mvn -Prun-it clean verify
 
 ## Changelog
 
-Refer to changelog for further details [changelog](CHANGELOG.md)
+Refer to the [project repository on GitHub][repo] for further details.
```

Comment:
If removing it, I would remove the entire section.  Its redundant.  This just says look at the repo and links back to itself.

Reply:
I was thinking if the repo was copied somewhere, the README would stay with it, but I think you're right.